public/index.html

```diff
@@ -9,7 +9,7 @@
       homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico?v=1">
```

Comment:
What is the point of this?

Reply:
Oh I changed the favicon from the default create react app favicon and the old one is cached for me. Favicon cache is kind of weird so it really needs cachebusting. I opened up a CRA issue to hopefully fix this there, however I'll add a comment for now.